package.json

```diff
@@ -39,15 +33,13 @@
     "compression": "^1.7.1",
     "copy-webpack-plugin": "^9.1.0",
     "cross-env": "^7.0.3",
-    "es6-promise": "^4.2.8",
     "eslint-plugin-header": "^3.1.1",
     "eslint-plugin-prettier": "^4.2.1",
     "esm": "^3.2.25",
     "express": "^4.17.2",
     "file-saver": "^2.0.5",
     "fs-extra": "^10.0.0",
     "golden-layout": "^1.5.9",
-    "html-loader": "^3.0.1",
```

Comment:
Not too sure about this one, but there is no reference about html anything in the webpack config